Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/configuration/ExchangeManagementPortDescriptor.java

```diff
@@ -0,0 +1,25 @@
+package gov.hhs.fha.nhinc.configuration;
+
+import gov.hhs.fha.nhinc.exchangemanagement.EntityExchangeManagementPortType;
+import gov.hhs.fha.nhinc.messaging.service.port.SOAP12ServicePortDescriptor;
+
+/**
+ *
+ * @author ttang
+ */
+public class ExchangeManagementPortDescriptor extends SOAP12ServicePortDescriptor<EntityExchangeManagementPortType> {
```

Comment:
Why we need this class at CONNECTCoreLib?

Reply:
ExchangeManagementPortDescriptor is required to make connection from adminGUI to the webservice; we don't wanted to moved this into exchangeManagement-service